src/apps/vpn/platforms/ios/ioscontroller.mm

```diff
@@ -217,29 +217,11 @@ emit statusUpdated(QString::fromNSString(serverIpv4Gateway),
 void IOSController::getBackendLogs(std::function<void(const QString&)>&& a_callback) {
   std::function<void(const QString&)> callback = std::move(a_callback);
 
-  QString groupId(GROUP_ID);
-  NSURL* groupPath = [[NSFileManager defaultManager]
-      containerURLForSecurityApplicationGroupIdentifier:groupId.toNSString()];
-
-  NSURL* path = [groupPath URLByAppendingPathComponent:@"networkextension.log"];
-
-  QFile file(QString::fromNSString([path path]));
-  if (!file.open(QIODevice::ReadOnly)) {
-    callback("Network extension log file missing or unreadable.");
-    return;
-  }
-
-  QByteArray content = file.readAll();
-  callback(content);
+  [IOSLoggerImpl getAppexLogsWithCallback:^(NSString* logs) {
+    callback(QString::fromNSString(logs));
+  }];
 }
 
 void IOSController::cleanupBackendLogs() {
```

Comment:
I noticed this was not getting called ever. I am not sure if that bug is related here, i should check it before merging. I think it's most likely a separate bug. I'll get back here with an answer.

Reply:
This is an unrelated bug. Filed https://mozilla-hub.atlassian.net/browse/VPN-5101